### PR TITLE
first pass update

### PR DESCRIPTION
**Pull request by stevemart**
* links
* minor formatting

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,11 +57,9 @@ Use the ``Deploy to IBM Cloud`` button **OR** create the services and run locall
 
 2. In Toolchains, click on Delivery Pipeline to watch while the app is deployed. Once deployed, the app can be viewed by clicking 'View app'.
 
-<p align="center">
-  <img width="600" src="doc/source/images/toolchain-pipeline.png">
-</p>
+![](doc/source/images/toolchain-pipeline.png)
```
Comment (scottdangelo) [inline README.md]: We've been adding a tag:
!["toolchain pipeline"].....

File: README.md [hunk 2]
```diff
@@ -76,43 +74,36 @@ Use the ``Deploy to IBM Cloud`` button **OR** create the services and run locall
...
+  * [**Watson Discovery**](https://cloud.ibm.com/catalog/services/discovery)
 
 ## 3. Configure Watson Discovery
 
 Fill in name you want to give to your service and click *Create*.
 
-<p align="center">
-  <img width="600" src="doc/source/images/discovery-1.png">
-</p>
+![](doc/source/images/discovery-1.png)
```
Comment (scottdangelo) [inline README.md]: tag

File: README.md [hunk 3]
```diff
@@ -155,22 +154,16 @@ $ yarn start
 On the **Trending News Panel**, click the RSS Feed button to launch a new tab in your browser. Cut and paste the tab URL into your favorite RSS Reader. If your RSS Feed Reader supports push notifications you can get alerted when trending topics change along with a news article for that topic.
 
 <p align="center">
-  <img width="400" src="doc/source/images/rss-2.png">
+  <img src="doc/source/images/rss-2.png">
```
Comment (scottdangelo) [inline README.md]: We've been moving away from HTML formatting in favor of pure markdown.

Comment (stevemart) [reply README.md]: these are long and narrow so I am keeping the <p align="center"> until i get to this step and redo the screenshots.

**Pull request merged**